docs/changes/newsfragments/7442.improved

```diff
@@ -1,2 +1,4 @@
-Exporting datasets to XArray no longer warns if two or more variables have setpoints of the same name but with different values.
-XArray automatically takes care of creating coordinates with names postfixed by `_n` to disambiguate these.
+Exporting datasets to XArray no longer warns if two or more variables are exported with different setpoints.
+No functionality of the export has changed
+If the setpoints have the same name missing values will be replaced by NaN and if different setpoints
+are used the exporter will automatically handle merging coordinates so each data variable is assigned its own coordinates.
```

Comment:
I think it's be useful to add a bit more explanation here. I suggest mentioning "values" of setpoints, not just setpoints, and I think "missing values" needs a bit of explanation, e.g. missing from where, from the "expected" grid, right? Also added punctuation for clarity, what do you think?
```suggestion
Exporting datasets to XArray no longer warns if two or more variables are exported with different setpoint values.
No functionality of the export has changed.
If the setpoints for different measured parameters have the same name, missing values will be replaced by NaN, and if different setpoint values
are used, the exporter will automatically handle merging coordinates so each data variable is assigned its own coordinates.
```